2019_amsterdam.html

```diff
@@ -0,0 +1,170 @@
+
+    <!-- Custom CSS -->
+    <link rel="stylesheet" href="css/creative.css" type="text/css">
+
+    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
+    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
+    <!--[if lt IE 9]>
+        <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
+        <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
+    <![endif]-->
+
+</head>
+
+<body id="page-top">
+
+    <nav id="mainNav" class="navbar navbar-default navbar-fixed-top">
+        <div class="container-fluid">
+            <!-- Brand and toggle get grouped for better mobile display -->
+            <div class="navbar-header">
+                <button type="button" class="navbar-toggle collapsed" data-toggle="collapse"
+                    data-target="#bs-example-navbar-collapse-1">
+                    <span class="sr-only">Toggle navigation</span>
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                </button>
+                <a class="navbar-brand page-scroll" href="index.html"><img src="img/about.png" class="navbar-img" />
+                    CoreDev Hacking</a>
+            </div>
+
+            <!-- Collect the nav links, forms, and other content for toggling -->
+            <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+                <ul class="nav navbar-nav navbar-right">
+                    <li>
+                        <a class="page-scroll" href="index.html#about">About</a>
+                    </li>
+                    <li class="active">
+                        <a class="page-scroll" href="pastevents.html">Past Events</a>
+                    </li>
+                    <li>
+                        <a class="page-scroll" href="2018_tokyo.html">Next Event</a>
+                    </li>
+                    <li>
+                        <a class="page-scroll" href="faq.html">FAQ</a>
+                    </li>
+                    <li>
+                        <a class="page-scroll" href="index.html#contact">Contact</a>
+                    </li>
+                </ul>
+            </div>
+            <!-- /.navbar-collapse -->
+        </div>
+        <!-- /.container-fluid -->
+    </nav>
+    <section class="bg-dark">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-8 col-lg-offset-2 text-center">
+                    <h1 class="section-heading">June 2019 Amsterdam</h1>
+                    <br />
+                    <h2>June 5-7 2019</h2>
+                    <p>CoreDev Hacking Meetings <em>(invite only)</em></p>
+                    <hr />
+                    <h4>June 8-9 2019</h4>
+                    <p><a href="https://breaking-bitcoin.com/">Breaking Bitcoin 2</a> (security conference on Bitcoin)
```

Comment:
Not sure how many of these we want to include, since they're not coredev events.

Reply:
+1 on avoiding non-core-dev events.

Reply:
Took out everything except coredev and the coworking space. Let me know if that works.